Bio/Affy/CelFile.py

```diff
@@ -81,35 +83,49 @@ def __init__(self):
         self.outliers = None
         self.modified = None
 
-import struct
-
 
 def read(handle, strict=False):
-    # If we fail to read the magic number, then it will remain None, and thus we will invoke read3 (if mode is not strict), or raise IOError if mode is strict.
+    # If we fail to read the magic number, then it will remain None, and thus
+    # we will invoke read3 (if mode is not strict), or raise IOError if mode is
+    # strict.
     magicNumber = None
-    # We check if the handle is a file-like object. If it isn't, and the mode is strict, we raise an error. If it isn't and the mode isn't strict, we continue (perhaps somebody has got a CEL-file-like iterable, which used to work with previous versions of biopython and we don't want to maintain backwards compatibility).
+    # We check if the handle is a file-like object. If it isn't, and the mode
+    # is strict, we raise an error. If it isn't and the mode isn't strict, we
+    # continue (perhaps somebody has got a CEL-file-like iterable, which used
+    # to work with previous versions of biopython and we don't want to maintain
+    # backwards compatibility).
     try:
         mode = handle.mode
-        # By definition an Affymetrix v4 CEL file has 64 as the first 4 bytes. Note that we use little-endian irrespective of the platform, again by definition.
+        # By definition an Affymetrix v4 CEL file has 64 as the first 4 bytes.
+        # Note that we use little-endian irrespective of the platform, again by
+        # definition.
         magicNumber = struct.unpack('<i', handle.read(4))[0]
         # reset the offset, to avoid breaking either v3 or v4.
         handle.seek(0)
     except (AttributeError, TypeError):
         if strict:
-            raise IOError("You have to pass a file-like object. You can get a file-like object by using `open`.")
+            raise IOError("You have to pass a file-like object. You can get "
+                          "a file-like object by using `open`.")
     if magicNumber != 64:
-        # In v4 we're always strict, as we don't have to worry about backwards compatibility
+        # In v4 we're always strict, as we don't have to worry about backwards
+        # compatibility
         if strict and mode != "r":
-            raise IOError("You're trying to open an Affymetrix v3 CEL file. You have to use a read mode, like this open(filename, \"r\").")
+            raise IOError("You're trying to open an Affymetrix v3 CEL file."
+                          "You have to use a read mode, like this"
+                          "`open(filename, \"r\")`.")
```

Comment:
You need spaces here at the string split points, e.g.:

``` python
            raise IOError("You're trying to open an Affymetrix v3 CEL file. "
                          "You have to use a read mode, like this "
                          "`open(filename, \"r\")`.")
```

However, it might be too strict to check for "r" only - using "rU" should work too.